components/eamxx/tests/single-process/ml_correction/ml_correction_standalone.cpp

```diff
@@ -43,7 +43,7 @@ TEST_CASE("ml_correction-stand-alone", "") {
   ad.initialize(atm_comm, ad_params, t0);
 
   const auto& grid = ad.get_grids_manager()->get_grid("Physics");
-  const auto& field_mgr = *ad.get_field_mgr(grid->name());
+  const auto& field_mgr = *ad.get_field_mgr();
```

Comment:
How was this not caught in the CI? Is this test not active?

Reply:
Only active on Livermore machines because we need to load the python ML package.  We didn't want to force all E3SM machines to make that available.